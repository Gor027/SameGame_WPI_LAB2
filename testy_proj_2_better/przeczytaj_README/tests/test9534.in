~~fff~ ~~~ ~   ff~ff ~ ~  ~ f f~~~~~ 
 ~~  f~~~  ~  f~f ~
 ~~  fffff~ f ~fff~~~~ff f~   ~~f f
 fff~  ~~~ ~ff  ~f ~ ~~ f f~ 
~f  ~ ~f~ ~ ~ f~~~f~ff~~  fff 
 ff~f  f   ~~f~   ff f  f~f~
  ~~~  ~  f   ~ ff~ 
~  f    f~ff      ~~fff~  ~ ~~fff~~ 
