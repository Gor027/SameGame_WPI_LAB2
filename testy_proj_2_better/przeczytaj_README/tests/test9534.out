      ~          f
~f    ~~       f ~f ~    ~
~~ff  f~  ~  f f~~~ ~    ~
~~f~  ~f~~~f ~ffff~ ~f~  f
ff~~~~f~f~~f f~~ff~~f~f  f~f f~~ ~
ff~ffff~~f~~f~f~~~fff~~fffff ~~f~f
f~f~f~~f~ff~f~~ff~~fff~f~~~~~fff~~~
